bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/framework/EquinoxContainer.java

```diff
@@ -154,6 +156,9 @@ private static void initConnectFramework(ModuleConnector moduleConnector, Equino
 		final File fwkStore = new File(configUrl.getPath());
 		@SuppressWarnings({"rawtypes", "unchecked"})
 		Map<String, String> config = (Map) equinoxConfig.getInitialConfig();
+		if (moduleConnector instanceof FrameworkUtilHelper) {
```

Comment:
I don't think this is necessary if you are adding during `init`.

Reply:
I was not really sure waht should be the very first time it is registered, e.g after creation of the factory? or after/before init? Or maybe when start is called?